samples/dnn/openpose.py

```diff
@@ -16,6 +16,7 @@
 parser.add_argument('--thr', default=0.1, type=float, help='Threshold value for pose parts heat map')
 parser.add_argument('--width', default=368, type=int, help='Resize input to specific width.')
 parser.add_argument('--height', default=368, type=int, help='Resize input to specific height.')
+parser.add_argument('--scale', default=0.003922, type=float, help='Scale for blob.')
```

Comment:
Please also remove redundant `To use Inference Engine backend, specify location of plugins:` at the beginning of this script.